app/model/AuditContext.scala

```diff
@@ -48,7 +48,6 @@ object RoutingReason {
   val IS_SELF_EMPLOYED = Reason("is-self-employed")
   val HAS_NINO = Reason("has-nino")
   val HAS_SA_UTR = Reason("has-sa-utr")
```

Comment:
@scottkwalker Shouldn't we remove the has-sa-utr, has-strong-credentials and has-only-one-enrolment? There values were introduced when we added the 2SV functionality. Also, we should remove them from allReasons list as well and update the unit and it tests.